tests/wallet/nft_wallet/test_nft_wallet.py

```diff
@@ -1251,7 +1252,7 @@ async def test_nft_set_did(two_wallet_nodes: Any, trusted: Any) -> None:
     for i in range(1, num_blocks):
```

Comment:
no need to create multiple blocks

Reply:
also, you need to fetch spend bundle from `nft_set_nft_did` and wait for it to get into mempool before continuing to create blocks, this applies to other calls that push spend bundles